release-calendar/src/client/index.html

```diff
@@ -6,6 +6,25 @@
   <title>AMP Release Calendar</title>
 </head>
 
+<style>
+body {
```

Comment:
Did you forget to move styles out of this file?

Reply:
Oh wait totally forgot to say. App.scss isn't imported into every tsx file and therefore it i make changes to body within app.sccs then I wont be changing the body and buttons styling everywhere i want it to be changed.